worlds/terraria/__init__.py

```diff
@@ -338,5 +338,9 @@ def check(state: CollectionState, location=location):
     def fill_slot_data(self) -> Dict[str, object]:
         return {
             "goal": list(self.goal_locations),
+            "achievements": self.multiworld.achievements[self.player].value,
+            "fill_extra_checks_with": self.multiworld.fill_extra_checks_with[
+                self.player
+            ].value,
```

Comment:
```suggestion
            "fill_extra_checks_with": self.multiworld.fill_extra_checks_with[self.player].value,
```
We normally do 120 chars per line in AP. While you are technically allowed to do that in a world, readability really suffers from it.